allcommands.ps1

```diff
@@ -46001,8 +46001,8 @@ function Install-DbaMaintenanceSolution {
 
             # Backup location
             if ($BackupLocation) {
-                $findBKP = 'C:\Backup'
-                $replaceBKP = $BackupLocation
+                $findBKP = 'SET @BackupDirectory     = NULL'
```

Comment:
Thanks so much for the PR! 😍 

I combine all changes to the allcommands.ps1 during publish so you do not have to worry about this particular file.

Reply:
Roger - just wanted to make sure. Will try and remember that ;)